styles/layouts/_main-navigation.scss

```diff
@@ -179,6 +179,15 @@
   }
 
 	#NavProfile {
+    .dropdown-trigger {
+      height: 40px;
+      width: 40px;
+      cursor: pointer;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+    }
```

Comment:
![pather](https://user-images.githubusercontent.com/9987732/201871959-1bf70a1d-7fc3-49d2-a4b9-9f0b082c10f0.jpg)

Reply:
so doing this shows the cursor pointer change but it doesnt open the dropdown sometimes, not sure why